opentelemetry-operator/chart/templates/smon.yaml

```
@@ -21,7 +21,7 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/component: opentelemetry-collector
-      app.kubernetes.io/instance: otel.metrics
+      app.kubernetes.io/instance: {{ .Release.Namespace }}.metrics
```

should we be a little bit more explicit here? 

it can be any arbitrary namespace, not sure if I would connect `.metrics` to otel.

Just an idea:

```suggestion
      app.kubernetes.io/instance: {{ .Release.Namespace }}.opentelemetry.metrics
```